apps/utr-backend/prisma/migrations/20220110115200_activate_post_gis/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- Adding the postGIS Extension to the database.
+CREATE EXTENSION IF NOT EXISTS postgis;
```

Comment:
Hast du geschaut ob das so auch bei Google Cloud funktioniert? Ansonsten haben wir vllt. Probleme, wenn wir unseren Deploy in Production ausführen und CREATE EXTENSION nicht unterstützt wird.

Reply:
Ich nehme an beim deployen in Production wird einmal `npx prisma migrate deploy` ausgeführt?
Dadurch würde dann ja die Migration in der postgreSQL Datenbank applyed werden.
`CREATE EXTENSION` ist ein Befehl der direkt von postgreSQL kommt und `IF NOT EXISTS` ist ein Parameter davon. Siehe [hier](https://www.postgresql.org/docs/current/sql-createextension.html).
Soweit ich das sehe müsste der auch von Google Cloud unterstützt werden. 
Auf deren [Doku](https://cloud.google.com/sql/docs/postgres/extensions) steht:
> Für die Verwendung einer Erweiterung müssen Sie diese installieren:
> - Führen Sie dazu im psql-Tool den Befehl CREATE EXTENSION aus.

und

> Für eine bestimmte PostgreSQL-Hauptversion können Sie im Befehl CREATE EXTENSION mithilfe der VERSION-Klausel eine PostGIS-Erweiterungsversion angeben.

Zwei Sachen die ich mir gerade vorstellen könnte, weswegen es Probleme machen könnte, wären dass die Migration nicht im psql-Tool ausgeführt wird und dass in der Cloud dann die Extension nicht dort kreiert werden kann, wo es ausgeführt wird.
Und das der postgres-User der die Migrations ausführt nicht die nötigen Rechte hat. Die benötigten Rechte haben nur User mit der Rolle `cloudsqlsuperuser`. Der postgres-Standarduser sollte die Rechte aber eigentlich haben.
Wie das dort in der Cloud aussieht müsstest du mir sagen. Lass uns morgen am besten nochmal drüber reden.